example/App.js

```diff
@@ -60,12 +60,17 @@ export default class App extends Component<Props> {
 
   search(text) {
     const phoneNumberRegex = /\b[\+]?[(]?[0-9]{2,6}[)]?[-\s\.]?[-\s\/\.0-9]{3,15}\b/m;
+    const emailAddressRegex = /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([^<>()[\].,;:\s@"]+\.)+[^<>()[\].,;:\s@"]{2,})$/i;
```

Comment:
do we really want to do this? aren't you only supposed to check if there's an `@` in the string

Reply:
It depends. Validating an email address accurately is actually fairly difficult. Email addresses are defined across a number of RFCs and don't really have a single well-defined grammar anywhere. If the goal is just differentiating versus a phone number, checking for @ is probably enough. Differentiating versus a contact name is a little harder. Anyway there are some decently accurate regular expressions here https://emailregex.com/ and a good explanation of the various rules and where they come from.

Reply:
ship it